backend/regalloc/regalloc_utils.ml

```diff
@@ -356,10 +367,21 @@ let remove_prologue_if_not_required : Cfg_with_layout.t -> unit =
   in
   if not prologue_required
   then
-    (* note: `Cfgize` has put the prologue in the entry block *)
-    let block = Cfg.get_block_exn cfg cfg.entry_label in
-    DLL.filter_left block.body ~f:(fun instr ->
-        match instr.Cfg.desc with Cfg.Prologue -> false | _ -> true)
+    (* note: `Cfgize` has put the prologue in the block after the entry block *)
```

Comment:
```suggestion
    (* note: `Cfgize` has put the prologue in the entry block or its successor. *)
```

Reply:
(Manually applied because of `ocamlformat`.)